packages/jest-test-sequencer/src/index.ts

```diff
@@ -219,9 +221,11 @@ export default class TestSequencer {
       if (test != null && !testResult.skipped) {
         const cache = this._getCache(test);
         const perf = testResult.perfStats;
+        const testRuntime =
+          perf.runtime ?? test.duration ?? perf.end - perf.start;
```

Comment:
fix is falling back to `perf.end - perf.start` (as it was before #9366), but also checking `test.duration` seems sensible to me.